src/types/mod.rs

```diff
@@ -11,7 +11,6 @@ mod capabilities;
 pub use self::capabilities::Capabilities;
 
 pub struct ZeroCopy<D> {
-    owned: Box<[u8]>,
```

Comment:
Wait, it is *definitely* not okay for this `Box` to be removed. It is what ensures that the underlying memory that `derived` points into doesn't get freed. So this particular change needs to be reverted. I'm guessing clippy claimed it wasn't used? Perhaps add a `#[allow` for the appropriate clippy lint here (+ a `#[cfg` so it builds even without clippy)?

Reply:
I've removed, but did not add the clippy exception. I should have seen the `unsafe` in that code and not removed the `owned`.

Reply:
I've removed, but did not add the clippy exception. I should have seen the `unsafe` in that code and not removed the `owned`.